streams/no.m3u

```diff
@@ -13,20 +13,40 @@ http://media1.adventist.no:1935/live/hope2/playlist.m3u8
 http://media1.adventist.no:1935/live/hope3/playlist.m3u8
 #EXTINF:-1 tvg-id="Kanal10Asia.se",Kanal 10 Asia (540p)
 http://cdn-kanal10.crossnet.net:1935/kanal10/kanal10asia/playlist.m3u8
-#EXTINF:-1 tvg-id="NRK2.no",NRK 2 [Geo-blocked]
-https://nrk-nrk2.akamaized.net/22/0/hls/nrk_2/playlist.m3u8
-#EXTINF:-1 tvg-id="NRK3.no",NRK 3 [Geo-blocked]
-https://nrk-nrk3.akamaized.net/23/0/hls/nrk_3/playlist.m3u8
-#EXTINF:-1 tvg-id="NRKSuper.no",NRK Super [Geo-blocked]
-https://nrk-nrksuper.akamaized.net/23/0/hls/nrk_super/playlist.m3u8
-#EXTINF:-1 tvg-id="TV2Nyhetsstrommen.no",TV2 Nyhetsstrømmen (720p)
-https://ws15-hls-live.akamaized.net/out/u/1153546.m3u8
-#EXTINF:-1 tvg-id="TV2Sport1.no",TV 2 Sport 1 (720p)
-https://ws31-hls-live.akamaized.net/out/u/1416253.m3u8
+#EXTINF:-1 tvg-id="NRK1.no",NRK 1 (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk1/muxed.m3u8

```

Comment:
https://nrk-live-no.akamaized.net/nrk1/muxed.m3u8

Reply:
I've handled this particular part now.

Reply:
Perfect, thank you.